recipes/magithub

```diff
@@ -0,0 +1,4 @@
+(magithub :fetcher github
+          :repo "vermiculus/magithub"
+          :files ("lisp/magithub.el"
```

Comment:
What would using the `:defaults` parameter look like? Something like below?

(magithub :fetcher github
          :repo "vermiculus/magithub"
          :files (:defaults
                  "LICENSE"))